### PR TITLE
Add support for additional exposure key fields

### PR DESCRIPTION
**Pull request by wescarr**

### CODE REVIEW

File: aggregate/aggregate.go [hunk 1]
```diff
@@ -163,14 +165,17 @@ func recordsToExposureKeys(rs records) []exposureKey {
...
-		if rollingStartNumber, err := strconv.ParseInt(r[1], 10, 64); err == nil {
-			keyData := r[0]
-
-			keys = append(keys, exposureKey{
-				keyData:            keyData,
-				rollingStartNumber: rollingStartNumber,
-			})
-		}
+		keyData := r[0]
+		rollingPeriod, _ := strconv.ParseInt(r[1], 10, 64)
```
Comment (joshgummersall) [inline aggregate/aggregate.go]: If these fail to parse, the zero value is okay to publish?

Comment (wescarr) [reply aggregate/aggregate.go]: Is there a nice way to do multiple error checks for each parseInt call?

Comment (joshgummersall) [reply aggregate/aggregate.go]: Of course not it’s Go

Comment (wescarr) [reply aggregate/aggregate.go]: 😂

**Comment by wescarr** [comment]
@joshgummersall Ok I think this is in good shape now

**Review by joshgummersall — APPROVED** [review]

**Pull request merged**